cmd/libs/go2idl/client-gen/generators/client_generator.go

```diff
@@ -217,6 +217,57 @@ func Packages(context *generator.Context, arguments *args.GeneratorArgs) generat
 	}
 	includedTypesOverrides := customArgs.IncludedTypesOverrides
 
+	// If there is a comment of the form "// +groupName=somegroup" or
```

Comment:
In lister-gen and informer-gen, rather than try to precompute the GVs in main.go, I do it in the `Packages` function, since that has access to `context.Universe`. We could restructure client-gen to do it that way, but that's a much much larger change.